src/Symfony/Bridge/Twig/Extension/EmojiExtension.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\Emoji\EmojiTransliterator;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+final class EmojiExtension extends AbstractExtension
+{
+    private static array $transliterators = [];
+
+    public function __construct(
+        private readonly string $defaultCatalog = 'slack',
```

Comment:
should we expose this configuration in TwigBundle ?

Reply:
I wondered. And I don't think so. Maybe because I'm a bit lazy 😅 ! But both catalogs are similar.

I don't know how people will use this filter, but passing the catalog each time is really not that complicated... and in my app, I use this filter a couple of time. So ...

Reply:
should be changed to "text" #54470 ;)